### ISSUE TITLE
Insert OR Extended Insert: NULL-value

### ISSUE THREAD

**Issue by GITMichaelK** [dbeaver/dbeaver#6874]
#### System information: 
- Operating system (distribution) and version: kUBUNTU 18.04.3
- DBeaver version 20190310

#### Connection specification:
- PostgreSQL JDBC Driver

#### Describe your question:
I need to insert lots of lines from a spreadsheet using the Dbeaver's ExtendedInsert function.
The table to be filled is defined as
```
CREATE TABLE "DateTime" (
	"ID" serial NOT NULL,
	"Date" date NOT NULL,
	"Begin" time NULL,
	"End" time NULL,
	"INt-val" int4 NULL,
	CONSTRAINT "ID_pkey" PRIMARY KEY ("ID")
);
```
There are lines in the spreadsheet, where the Begin and / or End cells are empty. 
Now I found, that I can ExtendedInsert
2019-10-01 and also 2019-10-01 TAB (for the empty column Begin)
having all empty cells filled with NULL after saving the line in DBeaver.
But ExtendedInsert
2019-10-01 TAB TAB (for the empty columns Begin and End) 
or
2019-10-01 TAB TAB 99 (for the empty columns Begin and End and 99 for INT-val)
fails, as BLANK (e.g. nothing) and not NULL is shown for the columns BEGIN and END.

How can I solve this issue, e.g. define "nothing" in the string to be inserted?
Thanks and regards, Michael

**Comment by GITMichaelK** [comment]
I tested this a bit more and found, that also simple pasting from existing lines having NULL-columns fails, so I think this is a bug.

**Comment by GITMichaelK** [comment]
@uslss: Found a [similar bug   @@here](https://github.com/dbeaver/dbeaver/issues/6026)

**Comment by uslss** [comment]
@GITMichaelK thanks :)
already fixed in scope of #6026 
ticket closed
